src/nvim/channel.h

```diff
@@ -85,7 +85,7 @@ struct Channel {
   bool callback_scheduled;
 };
 
-EXTERN PMap(uint64_t) *channels;
+extern PMap(uint64_t) *channels;
```

Comment:
`EXTERN PMap(uint64_t) *channels INIT(= NULL);`

Reply:
Thanks, will change it.